lib/oneview-sdk/resource/api2000/c7000/server_profile.rb

```diff
@@ -0,0 +1,22 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative '../../api1800/c7000/server_profile'
+
```

Comment:
i have already added this file in previous PR to pass the UTs. So have a look into this while merging to avoid conflicts

Reply:
its taken care.